packages/omg-js-childchain/src/childchain.js

```diff
@@ -42,16 +42,18 @@ class ChildChain {
   /**
   * Creates a ChildChain object
   *
-  * @param {Object} config the rootchain configuration object
+  * @param {Object} config the childchain configuration object
   * @param {string} config.watcherUrl the url of the watcher server (running in security-critical and informational mode)
   * @param {string} [config.watcherProxyUrl] *optional* the proxy url for requests made to the watcher server
+  * @param {string} config.verifyingContract the address of the Rootchain contract 
   * @return {ChildChain} a ChildChain Object
   *
   */
-  constructor ({ watcherUrl, watcherProxyUrl }) {
-    Joi.assert({ watcherUrl, watcherProxyUrl }, childchainConstructorSchema)
+    constructor ({ watcherUrl, watcherProxyUrl, verifyingContract }) {
```

Comment:
i would name this `plasmaContractAddress` to avoid the confusion of figuring out what is a verifyingContract, and for consistency with `rootChain`

`* @param {string} config.plasmaContractAddress the address of the PlasmaFramework contract`

Reply:
👍 given that there are multiple plasma contracts, I will make it explicit that we want `PlasmaFramework`